ext/plotting.jl

```diff
@@ -85,6 +85,121 @@ function Attractors.heatmap_basins_attractors!(ax, grid, basins, attractors;
     return ax
 end
 
+function colors_from_keys_shaded(ukeys)
+    # Unfortunately, until `to_color` works with `Cycled`,
+    # we need to explicitly add here some default colors...
+    COLORS =[ 
+        :black,
+        :red,
+        :green, 
+        :blue,
+        :purple,
+        :yellow,]
```

Comment:
Let's use the colors we already use please. There is a simple function that given `ukeys` it gives you colors. If you want to make these colors lighter or darker, use the function:
```
"""
    darken_color(c, f = 1.2)
Darken given color `c` by a factor `f`.
If `f` is less than 1, the color is lightened instead.
"""
function darken_color(c, f = 1.2)
    c = to_color(c)
    return RGBAf(clamp.((c.r/f, c.g/f, c.b/f, c.alpha), 0, 1)...)
end
```

This way, we also resolve the problem that we only have 6 colors, as the function I mention works for any number of attractors.